CHANGELOG.md

```diff
@@ -36,22 +41,22 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/) and this
 Minor documentation and bug fixes.
 
 ### Added
+
 - [`<Tab>`](https://github.com/illright/attractions/blob/master/attractions/tab/tab.svelte)s
   now also dispatch `click` event.
 
 ### Fixed
+
 - Mobile navigation in documentation ("Components" tab dropdown now works as expected).
 - Styles now load properly when navigating directly to a page in the docs (without passing through the main page first).
 - The `DatePicker` now loses focus properly.
 
-
 ## [2.1.0] - 2020-08-21
 
 First stable release with proper documentation.
 Previous, undocumented, releases can be found in [the releases section](https://github.com/illright/attractions/releases).
 
-
-[Unreleased]: https://github.com/illright/attractions/compare/v2.2.3...HEAD
+[unreleased]: https://github.com/illright/attractions/compare/v2.2.3...HEAD
```

Comment:
Is there a way to force this to be uppercased? I would like for the heading to say "Unreleased"

Reply:
It shouldn't matter, should it? I thought these anchors were case-insensitive

Reply:
Oh, ok then. I suspected that but didn't check

Reply:
Well, me neither